### ISSUE TITLE
Weird characters in terminal 

### ISSUE THREAD

**Issue by lafolle** [Ruin0x11/oxidoc#13]
Hi,

On issuing this command,  i get some weird characters in documentation: `oxidoc serde::de::DeserializeOwned`

```
(serde-1.0.27)
ESC[1m==== Trait serde::de::DeserializeOwnedESC[0m

----------

  pub trait DeserializeOwned { /* fields omitted */ }

----------

A data structure that can be deserialized without borrowing any data from
the deserializer.

This is primarily useful for trait bounds on functions. For example a
ESC[48;5;0;38;5;7mESC[0mESC[48;5;0;38;5;7mESC[0mESC[48;5;0;38;5;7mfrom_strESC[0mESC[48;5;0;38;5;7mESC[0mESC[48;5;0;38;5;7mESC[0m function may be able to deserialize a data structure that borrows
from the input string, but a ESC[48;5;0;38;5;7mESC[0mESC[48;5;0;38;5;7mESC[0mESC[48;5;0;38;5;7mfrom_readerESC[0mESC[48;5;0;38;5;7mESC[0mESC[48;5;0;38;5;7mESC[0m function may only deserialize
owned data.

ESC[48;5;0;38;5;7mESC[0mESC[48;5;0;38;5;7mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m#ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;182mESC[0mESC[48;5;0;38;5;182museESC[0mESC[48;5;0;38;5;182mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mserdeESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m::ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mdeESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m::ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m{ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mDeserializeESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m,ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m DeserializeOwnedESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m}ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m;ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;7mESC[0mESC[48;5;0;38;5;7mESC[0m
ESC[48;5;0;38;5;7mESC[0mESC[48;5;0;38;5;7mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m#ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;182mESC[0mESC[48;5;0;38;5;182museESC[0mESC[48;5;0;38;5;182mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mstdESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m::ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mioESC[0mESC[48;5;0;38;5;188mESC[0m
ESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m::ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m{ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mReadESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m,ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mResultESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m}ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188m;ESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;188mESC[0mESC[48;5;0;38;5;7mESC[0mESC[48;5;0;38;5;7m                   ESC[0mESC[48;5;0;38;5;7mESC[0m  
```
Can't figure out if the problem is with my terminal or oxidoc?

Thanks

**Comment by Ruin0x11** [comment]
Do you use Windows? Because if so it seems that the Markdown renderer I use only supports ANSI, and ANSI support was only introduced in Windows 10. Otherwise, let me know which terminal you use so I can investigate further.

**Comment by lafolle** [comment]
No,  I'm not on Windows but Linux: `Ubuntu 17.10`.
Using Gnome terminal with `TERM` being `screen-256color` (if that is helpful).

LKM if you need more info.

**Comment by lafolle** [comment]
So,  disabling pager produces correct output.
![oxidoc-13](https://user-images.githubusercontent.com/516298/38928178-9e0dc866-4325-11e8-9ea9-7aa10e1270c6.png)

Wondering if we can make pager optional as plain output to stdout should suffice most needs?

**Comment by Ruin0x11** [comment]
Sure, not a problem.

2018年4月18日(水) 4:08 Karan Chaudhary <notifications@github.com>:

**Comment by Ruin0x11** [comment]
Closed by #14.
